AndroidTestAutomationStarter/app/src/androidTest/java/com/novoda/androidstoreexample/tests/EspressoTestExampleWithRepositories.kt

```diff
@@ -0,0 +1,62 @@
+package com.novoda.androidstoreexample.tests
+
+import android.support.test.rule.ActivityTestRule
+import com.novoda.androidstoreexample.activities.MainActivity
+import com.novoda.androidstoreexample.data.repositories.ArticleRepository
+import com.novoda.androidstoreexample.userflows.productUserFlow
+import org.junit.BeforeClass
+import org.junit.Rule
+import org.junit.Test
+
+class EspressoTestExampleWithRepositories {
+
+    private val activityTestRule = ActivityTestRule<MainActivity>(MainActivity::class.java)
+
+
+    @get:Rule
+    var activityRule: ActivityTestRule<MainActivity> = activityTestRule
+
+    companion object {
```

Comment:
In Order to use BeforeClass you have to be able to use statics and this only works in Objects or Companion Objects